docs/getting-started/install-scylla/oss-install-aws.rst

```diff
@@ -0,0 +1,112 @@
+==========================
+Launch ScyllaDB on AWS
+==========================
+
+This article will guide you through self-managed ScyllaDB deployment on AWS. For a fully-managed deployment of ScyllaDB 
+as-a-service, see `ScyllaDB Cloud documentation <https://cloud.docs.scylladb.com/>`_.
+
+To deploy ScyllaDB, you can:
+
+* Launch instances from ScyllaDB AMI (GA)
+* Launch a ScyllaDB cluster using AWS CloudFormation (for evaluation only)
```

Comment:
Lest remove CloudFormation from here and from the section below.
We can bring it back if and when its prodcution ready.

Reply:
Fair enough. Let's keep it consistent with the rest of the doc.